video-snippets/share-over-search-bar.html

```diff
@@ -0,0 +1,380 @@
+    background-repeat: no-repeat;
+    cursor: pointer;
+    transform: scale(0.95);
+}
+
+#share-video-snippet-{{ snippet_id }} .twitter:hover {
+    transform: scale(1);
+    transform: translateY(-1px);
+    transition-property: transform;
+    transition-duration: 0.1s;
+    transition-timing-function: ease-out;
+}
+
+#share-video-snippet-{{ snippet_id }} .snippet-box {
+    position: relative;
+    background-color: rgba(255,255,255,0);
+    opacity:1;
+    background-position: 65%;
+    box-shadow: 0px 1px 1.5px rgba(0,0,0,0.15),
+    inset 0px 1px 0px rgba(255,255,255,1);
+    display: flex;
+    align-items: center;
+    transform: scale(1);
+}
+
+#share-video-snippet-{{ snippet_id }} .snippet-box:hover {
+    background-color: rgba(255,255,255,0.4);
+}
+
+#logo-replacement-{{ snippet_id }} {
+    position: relative;
+    top: 0px;
+    left: 10px;
+    margin: 22px auto 0 auto;
+    width: 666px;
+    height: 0;
+    overflow: visible;
+    pointer-events: none;
+    opacity: 0;
+    display: none;
+}
+
+#logo-replacement-{{ snippet_id }}.show {
+    display: block;
+    opacity: 1;
+    height: 310px;
+}
+
+#logo-replacement-{{ snippet_id }} .video-container {
+    width: 480px;
+    position: absolute;
+    left: 90px;
+    top: 15px;
+}
+
+#logo-replacement-{{ snippet_id }} .video-container video {
+    pointer-events: all;
+    cursor: pointer;
+    width: 480px;
+    height: 270px;
+}
+
+#logo-replacement-{{ snippet_id }} .video-container video.played {
+    cursor: auto;
+}
+
+#logo-replacement-{{ snippet_id }} .video-overlay {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+}
+
+#logo-replacement-{{ snippet_id }} .video-overlay .button {
+    display: none;
+    cursor: pointer;
+    background: none;
+    border: 0;
+    color: #FFF;
+    font-weight: bold;
+    font-size: 16px;
+    letter-spacing: -.05em;
+    text-transform: uppercase;
+    text-align: center;
+    margin: 75px auto 0;
+}
+
+#logo-replacement-{{ snippet_id }} .replay {
+    pointer-events: auto;
+}
+
+#logo-replacement-{{ snippet_id }} .video-overlay .button.show {
+    display: block;
+}
+
+#logo-replacement-{{ snippet_id }} .video-overlay .button::before {
+    display: block;
+    color: #FFF;
+    cursor: pointer;
+    line-height: 100px;
+    font-size: 42px;
+    text-align: center;
+    margin: 0 auto 20px;
+    border-radius: 50%;
+    background: #0C99D5;
+    border: 3px solid #B2E5FB;
+    height: 94px;
+    width: 94px;
+    transition: all 0.2s ease;
+    font-family: fontello-{{ snippet_id }};
+}
+
+#logo-replacement-{{ snippet_id }} .replay::before {
+    content: '\e800';  /* Replay icon */
+}
+
+#logo-replacement-{{ snippet_id }} .play::before {
+    content: '\e801';  /* Play icon */
+}
+</style>
+
+<div id="share-video-snippet-{{ snippet_id }}" class="snippet">
+  <div class="strings" data-tweet-text="{{ tweet_text|safe }}" data-tweet-hashtag="{{ tweet_hashtag|safe }}"></div>
```

Comment:
This can break is `tweet_text` or `tweet_hashtag` contains `"`. You can either escape it or move `tweet_text` into a hidden div